llvm/test/CodeGen/ARM/loopvectorize_pr33804.ll

```diff
@@ -74,6 +74,10 @@ for.body14.i.i:                                   ; preds = %for.body14.i.i, %en
   %loadf = load float, ptr %arrayidx15.i.i1427, align 4
   %next19.i.i = getelementptr inbounds %struct.CvNode1D, ptr undef, i32 %i.1424.i.i, i32 1
```

Comment:
While you are here, could you also replace the `undef` pointer operands here and elsewhere to make the test defined properly? The new stores are needed to avoid the loads being optimized away, which would be good to do separately.

Reply:
Sure!